### ISSUE TITLE
Medium cluster drawn bad (missing notable)

### ISSUE THREAD

**Issue by Pegoth** [PathOfBuildingCommunity/PathOfBuilding#655]
One of the notable not drawn at all of this item, if i change to "Adds 5 Passive Skills (enchant)" it will be drawn correctly, but will cost one additional travel points.

Passives in game: https://i.imgur.com/er0NEiD.png
Passives in PoB: https://i.imgur.com/bzze4c3.png
Passives in PoB with "Adds 5 Passive Skills (enchant)": https://i.imgur.com/DceDZ7y.png

Item in question:
```
Rarity: Rare
Entropy Stone
Medium Cluster Jewel
--------
Requirements:
Level: 54
--------
Item Level: 81
--------
Adds 4 Passive Skills (enchant)
Added Small Passive Skills grant: 6% increased effect of Non-Curse Auras from your Skills (enchant)
--------
Added Small Passive Skills also grant: +5 to Intelligence
Added Small Passive Skills also grant: +6 to Maximum Energy Shield
1 Added Passive Skill is Master of Command
1 Added Passive Skill is a Jewel Socket
--------
Place into an allocated Medium or Large Jewel Socket on the Passive Skill Tree. Added passives do not interact with jewel radiuses. Right click to remove from the Socket.
--------
Corrupted
```

**Comment by LocalIdentity** [comment]
This’ll be fixed in the next update

**Comment by Vulcaiiin** [comment]
![image](https://user-images.githubusercontent.com/20699669/79224657-c88c8f00-7e5b-11ea-8d79-c591cc336b99.png)

Have the same trouble.
As you can see on the previous image the cluster Jewel only show 3 nodes instead of Four, the nod Pure Might isn't showing, I don't know why it does happen. I've tried with other one and it works, this one in particularly does not work.
Version: 1.4.167.2

**Comment by LocalIdentity** [comment]
Fixed in 1.4.169.1
